contracts/libs/BLS.sol

```diff
@@ -114,7 +114,7 @@ library BLS {
         uint256[1] memory out;
 
         // prettier-ignore
-        uint256 precompileGasCost = BNPairingPrecompileCostEstimator(COST_ESTIMATOR_ADDRESS).gasCost(inputSize);
+        uint256 precompileGasCost = BNPairingPrecompileCostEstimator(COST_ESTIMATOR_ADDRESS).getGasCost(size + 1);
```

Comment:
This slip through from the previous PR, resulting 6x higher gas cost

where 
size = pubkeys.length
inputSize = (size + 1) * 6